bio/varscan/somatic/wrapper.py

```
@@ -6,45 +6,30 @@
 __license__ = "MIT"
 
 
-import os.path as op
+import os
```

Ruff (0.8.2)

`os` imported but unused

Remove unused import: `os`

(F401)